src/components/Candidate.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import styled from 'styled-components';
+import { url } from './url';
+import axios from 'axios';
+
+const Candidate = ({ candidateName, voteCount, id, getCandidates }) => {
+  const getVote = async (id) => {
+    try {
+      const vote = await axios.get(`${url}/vote`, {
+        params: {
+          id: id,
+        },
+      });
+      getCandidates();
```

Comment:
누구에게 투표했는지도 alert해주면 좋을 것 같아요!

```suggestion
      getCandidates();
      alert(`${candidateName}님께 투표완료!`)
```